server/observable.go

```diff
@@ -809,6 +830,25 @@ func (o *Observable) incDeliveryCount(sseq uint64) uint64 {
 	return o.rdc[sseq] + 1
 }
 
+func (o *Observable) notifyDeliveryExceeded(sseq, dcount uint64) {
+	e := &ObservableDeliveryExceededEvent{
+		Schema:     "io.nats.jetstream.event.v1.max_deliver",
```

Comment:
I think some comments/guidance around schema would be useful.

`<Product>.<system>.<type>.<version>.<schema-name>`

e.g. product will likely always be `io.nats`,  subsystems today are `jetstream` and then `server`.  Can later add new schemas for bridges and other components.  What else do you see alongside `event` that would require a schema?   `command`, maybe `message` for user messages as the schema is offered as a feature?

Reply:
Already I consider these 2 different kind of events. One is a observability sample for monitoring and is an event that forms the basis of building DLQ and admin interfaces. 

I wasn’t sure though what to call these 2 different things but they are different for sure. The DLQ related one is an event - the other something else.

Reply:
I made the subjects be like `$JS.SAMPLE.OBS_ACK.<set>.<obs>` and `$JS.EVENT.MAX_DELIVER.<set>.<obs>`

and renamed the `type` of the sample to `sample`, this way we'll get all events published to `$<system>.<type>.<schema name>` and so multiple versions can go in the same subject which seems right, and there's a clear relation between these 2 leaving it discoverable.

This also means we can write ACLs so Bob can only get events from Bobs stream - and monitoring dont also get events etc. We'll make it so a observable can filter on a wildcard so one can make an observable on a set that holds all events to retrieve only those for one set etc.